clean.sh

```diff
@@ -1,16 +1,12 @@
 #! /bin/sh
 
-set -e
```

Comment:
Since we are removing this, we might as well remove the `|| :` sections as well.

Reply:
Good point, thanks. I will remove that.